sdk/tracing/src/test/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessorTest.java

```diff
@@ -171,15 +172,23 @@ void exportMoreSpansThanTheBufferSize() {
     ReadableSpan span4 = createSampledEndedSpan(SPAN_NAME_1);
     ReadableSpan span5 = createSampledEndedSpan(SPAN_NAME_1);
     ReadableSpan span6 = createSampledEndedSpan(SPAN_NAME_1);
-    List<SpanData> exported = waitingSpanExporter.waitForExport();
-    assertThat(exported)
-        .containsExactly(
-            span1.toSpanData(),
-            span2.toSpanData(),
-            span3.toSpanData(),
-            span4.toSpanData(),
-            span5.toSpanData(),
-            span6.toSpanData());
+
+    // Give time for the BatchSpanProcessor to attempt to export spans.
+    Thread.sleep(500);
```

Comment:
I generally don't like `Thread.sleep` in tests but this is the best I could come up with here.

Reply:
is there a way to use awaitility to get rid of this? I assume there isn't anything obvious or you would have done it. :)

Reply:
Couldn't find anything myself, was hoping someone else would find it but this may be best...